index.js

```diff
@@ -30,10 +30,11 @@ var colors = {
 
 module.exports = Nanologger
 
-function Nanologger (name) {
-  if (!(this instanceof Nanologger)) return new Nanologger(name)
+function Nanologger (name, colors) {
```

Comment:
Yes, great idea. I added the `opts` object as well as it's assertion. Therefore I added `assert`. I hope this is okay for you. 👍